src/components/MovieDetails/MovieDetails.scss

```diff
@@ -1,12 +1,53 @@
 .MovieDetails {
-  display: grid;
+  display: flex;
+  align-items: center;
+}
+
+.movie-info {
   justify-content: center;
+  margin: 5% auto;
+  width: 45%;
+  height: 45%;
```

Comment:
Can we use hard values for height instead of percentages? If we do that, then as the browser window shrinks, the content will only shrink on the horizontal axis rather than both horizontal and vertical axes.